internal/ingress/controller/controller.go

```diff
@@ -221,17 +223,12 @@ func (n *NGINXController) CheckIngress(ing *networking.Ingress) error {
 		return nil
 	}
 
-	if !class.IsValid(ing) {
```

Comment:
Why did we do this? This effectively means one e.g can not deploy two ingerss-nginx instances in a cluster where one rejects catch-all ingresses while the other does not. This change will mean all catch-all ingresses in the cluster are rejected.